README.md

```diff
@@ -340,6 +340,10 @@ d.memory["main_arena"] = b"12345678"
 
 `wait()` will block the execution of the Python script until the debugging process interrupts.
 
+`finish([exact=True])` will continue the execution until the current function returns or the process is stopped (e.g., breakpoint).
+
+In **exact** mode, the debugger will step until the return instruction of the current frame is executed. In **non-exact** mode, a breakpoint is placed on the return address and execution continues. Non-exact mode is faster but can fail in case of non-standard control flow (e.g., packing). Both will return on breakpoint.
```

Comment:
I don't think we're gonna support ARMv7 soon...
IIRC this feature works on aarch64, so I don't see the need for clarification

Reply:
A software step consists of a series of conts and breakpoints. Without specifying that the exact mode is based on a feature available in the hardware of supported architectures, the difference between it and the heuristic mode may not be immediately clear. There are no references to the hardware nature of the exact mode. Users need to understand what makes the exact mode "exact" to properly decide when to use the exact mode versus the heuristic mode. Full details are not necessary, but information about the hardware nature is important IMO, similar to the distinction between software and hardware breakpoints.

Reply:
"In **exact** mode, the debugger will iteratively perform single hardware
teps until the return instruction of the current frame is executed. In **non-exact** mode, a breakpoint is placed on the return address and execution continues. In most cases, when the function frame is aligned with the calling convention, the flow of both modes coincides. However, in case of packed or non-conventional binaries, it is possible that the address found on the stack is wrong or cannot be detected by a software breakpoint. The reason why non-exact mode could be preferred is speed, as allowing execution to continue is still faster than hardware stepping for big functions."

Is this enough?